macOS/DuckDuckGo/NetworkProtection/ExcludedDomains/ExcludedDomainsViewController.swift

```diff
@@ -99,7 +99,11 @@ final class ExcludedDomainsViewController: NSViewController {
     }
 
     @IBAction func addDomain(_ sender: NSButton) {
-        AddExcludedDomainView(title: UserText.vpnAddExcludedDomainTitle, buttonsState: .compressed, cancelActionTitle: UserText.vpnAddExcludedDomainCancelButtonTitle, cancelAction: { dismiss in
+        addDomain()
+    }
+
+    func addDomain(domain: String = "") {
+        AddExcludedDomainView(title: UserText.vpnAddExcludedDomainTitle, domain: domain, buttonsState: .compressed, cancelActionTitle: UserText.vpnAddExcludedDomainCancelButtonTitle, cancelAction: { dismiss in
```

Comment:
Diff looks awful, but I'm basically allowing the caller to specify which site to add (so it shows up in the input text field).